### PR TITLE
fixes #33

### LINKED ISSUE #33: profile page has duplicate buttons
When looking at your own profile

screen shot 2015-06-17 at 7 06 07 pm

### PR DESCRIPTION
**Pull request by CosmicWebServices**
Tested
- Andrew Rodebaugh

### CODE REVIEW

**Comment by CosmicWebServices** [comment]
Just tested it and it fixes #33

**Comment by CosmicWebServices** [comment]
@themightychris can you look at this?

File: html-templates/people/person.tpl [hunk 1]
```diff
@@ -13,16 +13,12 @@
...
 
                 <header class="page-header">
-                    <div class="header-buttons">
-                    {if $.User->ID == $Person->ID || (ProfileRequestHandler::$accountLevelEditOthers && $.User->hasAccountLevel(ProfileRequestHandler::$accountLevelEditOthers))}
-                        <a class="button" href="/profile{tif $.User->ID != $Person->ID ? cat('?person=', $Person->ID)}">Edit Profile</a>
-                    {/if}
-                    </div>
-
                     <h2 class="header-title">{$Person->FullName|escape} <small class="muted">Public Feed</small></h2>
                     {if $ownProfile}
```
Comment (themightychris) [inline html-templates/people/person.tpl:17]: only having the edit button within the `$ownProfile` check prevents admins from seeing it for other users

**Review by themightychris — COMMENTED** [review]
changes made and merged

**Pull request merged**